milestones/3_human_brain_mapping_journal_submission/experiments/ex_2_pli_generation_optimization/example_pli_analysis.m

```diff
@@ -0,0 +1,214 @@
+%% Yacine Mahdid 17 Septembre 2020
+
+function example_pli_analysis(p_id, epoch)
+    %% EXAMPLE PLI ANALYSIS this is an example script to run the pli generation as fast as possible
+    % input:
+    % p_id: is the id of the participant (e.g. MDFAXX)
+    % epoch: is the epoch of interest (e.g. eyesclosed_1)
+
+    %% Path Setup
+    % Local Source
+    %{
+    INPUT_DIR = "/media/yacine/My Book/datasets/consciousness/AEC vs wPLI/source localized data/";
+    OUTPUT_DIR = "/media/yacine/My Book/test_result/ex_1_pli_generation_comparison/pli_surrogate_graphs/";
+    %}
+
+    % Remote Source
+    %
+
+    % Create a "local" cluster object
+    distcomp.feature( 'LocalUseMpiexec', false ); % This was because of some bug happening in the cluster
+    local_cluster = parcluster('local');
+
+    % Modify the JobStorageLocation to $SLURM_TMPDIR
+    local_cluster.JobStorageLocation = strcat('/scratch/yacine08/', getenv('SLURM_JOB_ID'));
+
+    % Start the parallel pool
+    parpool(local_cluster, str2num(getenv('SLURM_CPUS_ON_NODE')))
+
+    INPUT_DIR = "/lustre03/project/6010672/yacine08/aec_vs_pli/data/source_localized_data/";
+    OUTPUT_DIR = "/lustre03/project/6010672/yacine08/aec_vs_pli/result/ex_2_pli_generation_optimization/";
+    NEUROALGO_PATH = "/lustre03/project/6010672/yacine08/NeuroAlgo";
+
+    % Add NA library to our path so that we can use it
+    addpath(genpath(NEUROALGO_PATH));
+
+    %}
+
+    % indice of the scalp regions
+    SCALP_REGIONS = [82 62 54 56 58 60 30 26 34 32 28 24 36 86 66 76 84 74 72 70 88 3 78 52 50 48 5 22 46 38 40 98 92 90 96 94 68 16 18 20 44 83 63 55 57 59 61 31 27 35 33 29 25 37 87 67 77 85 75 71 73 89 4 79 53 51 49 6 23 47 39 41 99 93 91 97 95 69 17 19 21 45];
+    NUM_REGIONS = length(SCALP_REGIONS);
+
+    % wPLI Parameters:
+    % Alpha bandpass
+    low_frequency = 8;
+    high_frequency = 13;
+
+    % Size of the cuts for the data
+    window_size = 10; % in seconds
+    step_size = 5; % in seconds
+    cut = 10;
+
+    % Type of graph to calculate
+    graph = 'wpli';
+
+    %% Setup the Directory Structure
+    mkdir(OUTPUT_DIR)
+
+    %% Calculate wPLI on all windows            
+    fprintf("Analyzing participant '%s' at epoch '%s'\n", p_id, epoch);
+
+    participant_in_path = strcat(INPUT_DIR, p_id, filesep, p_id, '_', epoch, '.mat');
+    participant_out_path = strcat(OUTPUT_DIR, p_id, '_', epoch, '_', graph, '.mat');
+
+    %% Load data
+    load(participant_in_path);
+
+    Value = Value(SCALP_REGIONS,:);
+    Atlas.Scouts = Atlas.Scouts(SCALP_REGIONS);
```

Comment:
Could you please specify what you are doing here? What is Atlas/ is it a special kind of Atlas ?

Reply:
Yes it's the AAL I've added more information in the comments